log-collector-script/linux/README.md

```diff
@@ -36,6 +36,8 @@ OPTIONS:
 
    --ignore_metrics Variable To ignore prometheus metrics collection; Pass this flag if DISABLE_METRICS enabled on CNI
 
+   --ignore_imds Variable To ignore IMDS based collection; Pass this flag if running on an EKS Hybrid node
```

Comment:
Can we use a more generic name here, if the intention is to support usage outside EC2? `--non-ec2`, `--outside-ec2`, or maybe just `--eks-hybrid` ?

Reply:
Done